tests/Univapay/Integration/Requests.php

```diff
@@ -242,11 +242,15 @@ public function createValidRefund()
         )->awaitResult(5);
     }
 
-    public function createValidSubscription($authorized = null, DateInterval $captureAfter = null)
-    {
+    public function createValidSubscription(
+        $authorized = null,
+        DateInterval $captureAfter = null,
+        TokenType $type = null
+    ) {
         $this->deactivateExistingSubscriptionToken();
+        $type = isset($type) ? $type : TokenType::SUBSCRIPTION();
```

Comment:
Could be `$type ?? TokenType::SUBSCRIPTION();`

Reply:
Is that syntax supported on php7?